docs/building-apps/example-application-tutorial/anchor-integration/setup.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: "Setup for Anchored Assets"
+sidebar_position: 10
+---
+
+An anchor is a Stellar-specific term for the on and off-ramps that connect the Stellar network to traditional financial rails, such as financial institutions or fintech companies. When a user deposits with an anchor, that anchor will credit their Stellar account with the equivalent amount of digital tokens. The user can then hold, transfer, or trade those tokens just like any other Stellar asset. When a user withdraws those tokens, the anchor redeems them for cash in hand or money in the bank. Read more about anchors in our [Anchor Documentation](https://developers.stellar.org/docs/anchoring-assets/overview).
+
+When a customer downloads a wallet application that is connected to an anchor service, their Stellar account can either be created by the wallet application or the anchor service. In this example, the account has been created by the wallet application, BasicPay. Account creation strategies are described more in-depth [here](../../application-design-considerations#account-creation-strategies).
+
+In this example, we’ll use an anchor on Stellar’s Testnet to simulate a bank transfer into and out of the user’s wallet using [SEP-6: Deposit and Withdrawal API](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0006.md) and/or [SEP-24: Hosted Deposit and Withdrawal](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0024.md).
+
+:::info
+
+SEPs define community-decided standards for interoperability on Stellar. Read more in our [SEPs section](../../../fundamentals-and-concepts/stellar-ecosystem-proposals).
+
+:::
+
+Our integrations will also use the following SEPs:
+
+- [SEP-1: Stellar TOML](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0001.md) - a file that provides a place for the Internet to find information about an organization’s Stellar integration
+- [SEP-9: Standard KYC Fields](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0009.md) - defines a list of standard KYC fields for use in Stellar ecosystem protocols
+- [SEP-10: Stellar Web Authentication](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0010.md) - defines the standard way for clients to create authenticated web sessions on behalf of a user who holds a Stellar account
+- [SEP-12: KYC API](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0012.md) - defines a standard way for Stellar clients to upload KYC information to anchors
+
+## Setup for anchored assets
+
+First, we need our application to know how it can communicate with anchors to get asset and infrastructure information. Do this by first querying an anchor’s stellar.toml file.
```

Comment:
How does this TOML relate to the SEP-1 section after this one? Can we move this to the SEP-1 section?

Reply:
It doesn't really. I just didn't write that very well lol. It's more about querying the network to see which assets have a home domain, but I got things mixed up.

Reply:
HMMM. Ok. I'm still wondering if we can move this whole section: 

"First, we need our application to know how it can communicate with anchors to get asset and infrastructure information. Do this by first querying an anchor’s stellar.toml file.

BasicPay takes care of all the anchor transfer details on the /dashboard/transfers page. See it in action here: https://basicpay.pages.dev/dashboard/transfers"

to the SEP-1: Stellar TOML section?